keps/sig-node/3960-pod-lifecycle-sleep-action/README.md

```diff
@@ -331,8 +337,21 @@ The change is opt-in, it doesn't impact already running workloads.
 
 ###### What specific metrics should inform a rollback?
 
+Metric `sleep_action_terminated_early_total` will be added in beta. 
```

Comment:
Can't comment on the unchanged line, so commenting here:

> Are there any tests for feature enablement/disablement?

Can you please update this question? Were tests created?

Reply:
Updated in second commit.